css/components/avatar.css

```diff
@@ -1 +1,54 @@
-/*avatar css*/
\ No newline at end of file
+/*
+|| avatar css
+*/
+
+/* Main Avatar */
+.avatar{
+    display: flex;
+	align-items: center;
+	justify-content: center;
+    flex-wrap: wrap;
+    margin: 0.3rem;
```

Comment:
You can use gap instead of margin: margin adds on all the sides of avatar, where as gap works only between flex-children. if you are using margin to center image, use padding as it will be part of the content